src/app/view/components/dot-theme-selector-dropdown/dot-theme-selector-dropdown.component.scss

```diff
@@ -52,3 +56,51 @@
         }
     }
 }
+
+dot-searchable-dropdown {
+    width: 100%;
+    display: block;
+}
+
+.searchable-dropdown__search {
+    display: flex;
+    margin: 0 $spacing-2;
+    position: relative;
+
+    dot-site-selector {
+        margin-right: $spacing-2;
+        display: block;
+
+        ::ng-deep {
+            .site-selector__title {
```

Comment:
Please add a comment here lilke: "This is to style as disabled textfield the label when there is only one site"